lib/lpc55-rot-startup/build.rs

```diff
@@ -39,3 +42,21 @@ fn gen_memory_range(
 
     Ok(())
 }
+
+fn gen_image_flash_range(
+    out_dir: &std::path::PathBuf,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let toml = build_util::env_var("HUBRIS_FLASH_OUTPUT")?;
+    let region: Region = toml::from_str(&toml)?;
+    let mut cfg = File::create(out_dir.join("config.rs")).unwrap();
+
+    writeln!(
+        cfg,
+        "use core::ops::Range;\n\n\
+             pub const IMAGE_FLASH: Range<usize> = {:#x}..{:#x};",
```

Comment:
Could we make this a `u32` to avoid the extra `try_into`?

Reply:
Absolutely: [b15deba](https://github.com/oxidecomputer/hubris/pull/1500/commits/b15deba04ff15e657111c53f21ab1d8b2df6d1da)